atlp-rwanda/ecommerce-app-legends-bn
# #38 feature for sending notifications

#### Description of Task to be completed?

- To send email notifications  to the product lifecycle changes
- To send app notifications due to the product lifecycle changes 

#### How should this be manually tested?

- git clone from repo
- switch into branch
- run npm i
- run npm run dev
- - try to add product
- check your email
- to test app notification:
- add html file into src folder and insert these codes into it:
-  

> <!DOCTYPE html>
> <html>
> <head>
>   <meta charset="UTF-8">
>   <title>Socket.IO Notifications Test</title>
>   <script src="https://cdn.socket.io/4.4.1/socket.io.min.js"></script>
> </head>
> <body>
>    <h1>Socket.IO Notifications Test</h1>   
>   <ul id="notifications"></ul>
>   <script>
>     const socket = io('http://localhost:5000');
>     socket.on('notification', (data) => {
>       const li = document.createElement('li');
>       li.textContent = data.message;
>       document.getElementById('notifications').appendChild(li);
>     });
>   </script>
> </body>
> </html>


#### What are the Trello stories?

- number 38
